docs/changes/table/14966.bugfix.rst

```diff
@@ -0,0 +1,4 @@
+Fixed issue #14964 that when grouping a Table on a mixin column such as ``Quantity`` or
```

Comment:
I think I'd write this more positively, as "Ensure that when ... the grouped table keys reflect the original column values. Before this fix, for...."